test/ffmpeg-qsv/vpp/denoise.py

```diff
@@ -38,9 +38,12 @@ def check_metrics(self):
 
     def compare(k, ref, actual):
       assert ref is not None, "Invalid reference value"
-      assert abs(ref[-3] - actual[-3]) < 0.2, "Luma (Y) out of baseline range"
-      assert abs(ref[-2] - actual[-2]) < 0.2, "Cb (U) out of baseline range"
-      assert abs(ref[-1] - actual[-1]) < 0.2, "Cr (V) out of baseline range"
+      effects = self.caps.get("effect", [ "Luma", "Chroma"])
+      if "Luma" in effects:
```

Comment:
Luma should always be supported until there is evidence claiming otherwise.  So let's always check Luma unconditionally.

Reply:
Yes , you are right.